### DISCUSSION TITLE
Request to Add Social Media Icons to Hero Icons Library

### DISCUSSION THREAD

**Discussion by AayushMohan** [tailwindlabs/heroicons#982] [Ideas]
Hey There,

As a long-time user of Hero Icons, I would like to kindly request the addition of social media icons to the library. This would make it more convenient for us to access social media icons without relying on third-party websites. I believe this addition would greatly benefit the Hero Icons community and enhance the versatility of this popular icon library.

Thanks!
Aayush Mohan

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
